### DISCUSSION TITLE
Allow all traffic through the firewall

### DISCUSSION THREAD

**Discussion by UMHB-InfoSec** [Security-Onion-Solutions/securityonion#5910] [Unsupported Versions] [answered]
I have a hardware firewall that is protecting access to my SO cluster and I am having a devil of a time allowing custom ports through the local FW on the SO boxes.

My SO cluster is on a dedicated and isolated subnet so if the traffic makes it through the FW I want the boxes to get it.

Is it possible just to disable the FW that is on the SO boxes since it is not really being used?  I see there seems to be some chain routing for the docker containers which means simply disabling iptables is probably not an option, but could someone help me setup a default permissive rule?

**Comment by hacker0ni** [comment, marked as answer]
I would highly suggest not doing this, having an extra layer of protection is always good. But for the sake of helping:
You can use the`so-firewall addhostgroup every_ip` to create a hostgroup and then include every IP address out there in that hostgroup using `so-firewall includehost every_ip 0.0.0.0`. Then all you'd need to do is to go to your minion.sls file for each node in your deployment and append the following (these files are in `/opt/so/saltstack/local/pillar/minions/<minion_name>.sls` on the manager node:
```
firewall:
  assigned_hostgroups:
    chain:
      DOCKER-USER:
        hostgroups:
          every_ip:
            portgroups:
              - portgroups.all
```
If this does not work you might have to go through each portgroup located in `/opt/so/saltstack/default/salt/firewall/portgroups.yaml` and add those portgroups to the last line with the same syntax like: `- portgroups.beats_5044`. After you add these portgroups, you need to apply them to all nodes using `sudo salt "*" state.apply firewall`

I have not tested this method, for my own sake. I accept no responsibility and only do this at your own risk.

**Reply by UMHB-InfoSec** [reply]
I apologize for not responding sooner.  Thank you for your response.

I agree that disabling the local FW is against the defense-in-depth paradigm, but in this instance I'm okay with it.

**Comment by Acewiza** [comment]
"Is it possible just to disable the FW that is on the SO boxes?"  Certainly.  Might not be a bad troubleshooting step.
